README.md

```diff
@@ -2,9 +2,10 @@
 
 # My-Portfolio
 
-> One paragraph statement about the project.
+This is My personal portfolio
+It describes my work aa s software developer.
 
```

Comment:
- [ ] Great job Your README is looking professional but you still have one thing to improve. Kindly I would recommend you to:
- Add the `Getting started` section that should explain how someone else can run or get the project in his/her local environment. you can include something like:
```
To get the content of this project locally you need to run this command in your terminal:
- `git clone your link
- `cd project name
```
- Add the more contacts in the author section like:
* Twitter: [name](https://profile link)
* LinkedIn: [name](https://profile link)